uml/sequence_diagrams/ClockView.txt

```diff
@@ -0,0 +1,46 @@
+@startuml
+participant DeactivatedClock 
+participant ClockView 
+participant BlueTooth
+participant Battery
+participant DisplayError 
+participant DisplayViews 
+
+opt Activation

```

Comment:
If everything below only happens if the Activation transition happens, I think it all needs to be wrapped in the opt.

Reply:
That's implicit because if the Activation transition doesn't happen , no transition from ClockView could happen.

Reply:
I think that for the part in the loop to happen either one of the optional transitions to ClockView must happen (but not both, which I think could happen if both are opt). 

I think something like this makes it clearer:
```
alt Watch==Deactivated
DeactivateWatch -> ClockView : Activate Watch
else Watch==Activated
DisplaySelection --> ClockView : Transition from Display Selection
end
```

What do you think ?

Reply:
ok fixed

Reply:
I don't think they need to be optional in the alt

Reply:
I think so, to indicate an optional action from the user. Otherwise the the requirement would be misleading.

If you are deactivated you don't automatically go to activated state (as would be implied without opt) and if activated you don't necessary move to clockview from DisplaySelection.

I don't repeat the opt when you receive an event to change state later in the diagram.